docs/source/contributor-guide/governance.md

```diff
@@ -38,6 +34,84 @@ As much as practicable, we strive to make decisions by consensus, and anyone in
+| Kazuyuki Tanimura       | kazuyukitanimura | [kazuyukitanimura](https://github.com/kazuyukitanimura) |             | Committer |
+| Eduard Karacharov       | korowa           | [korowa](https://github.com/korowa)                     |             | Committer |
+| Siew Kam Onn            | kosiew           | [kosiew](https://github.com/kosiew)                     |             | Committer |
+| Lewis Zhang             | linwei           |                                                         |             | Committer |
+| Matt Butrovich          | mbutrovich       | [mbutrovich](https://github.com/mbutrovich)             |             | Committer |
+| Metehan Yildirim        | mete             |                                                         |             | Committer |
+| Marko Milenković        | milenkovicm      | [milenkovicm](https://github.com/milenkovicm)           |             | Committer |
+| Wang Mingming           | mingmwang        | [mingmwang](https://github.com/mingmwang)               |             | Committer |
+| Michael Ward            | mjward           |                                                         |             | Committer |
+| Marco Neumann           | mneumann         | [crepererum](https://github.com/crepererum)             |             | Committer |
+| Zhong Yanghong          | nju_yaho         |                                                         |             | Committer |
+| Paddy Horan             | paddyhoran       | [paddyhoran](https://github.com/paddyhoran)             |             | Committer |
+| Parth Chandra           | parthc           | [parthchandra](https://github.com/parthchandra)         |             | Committer |
+| Rémi Dettai             | rdettai          | [rdettai](https://github.com/rdettai)                   |             | Committer |
+| Sun Chao                | sunchao          | [sunchao](https://github.com/sunchao)                   |             | Committer |
+| Daniel Harris           | thinkharderdev   | [thinkharderdev](https://github.com/thinkharderdev)     |             | Committer |
+| Raphael Taylor-Davies   | tustvold         | [tustvold](https://github.com/tustvold)                 |             | Committer |
+| Weijun Huang            | weijun           | [Weijun-H](https://github.com/Weijun-H)                 |             | Committer |
+| Yang Jiang              | yangjiang        | [Ted-jiang](https://github.com/Ted-jiang)               |             | Committer |
+| Yijie Shen              | yjshen           | [yjshen](https://github.com/yjshen)                     |             | Committer |
+| Yongting You            | ytyou            | [2010YOUY01](https://github.com/2010YOUY01)             |             | Committer |
+| Qi Zhu                  | zhuqi            | [zhuqi-lucas](https://github.com/zhuqi-lucas)           |             | Committer |
```

Comment:
@zhuqi-lucas  would you like to add a affiliation?

Reply:
Thank you @alamb , i added my affiliation now.